.github/workflows/dart_ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [ stable, 2.13.4, dev ]
+        sdk: [ stable, 2.13.4 ]
```

Comment:
I removed the dev branch of Dart since that's now 3.0 dev and it immediately fails due to the SDK constraints of this package.